draft-ietf-httpbis-binary-message.md

```diff
@@ -154,8 +154,9 @@ length-prefixed value. Both name and value are sequences of bytes that cannot
 be zero length.
 
 The format allows for the message to be truncated before any of the length
-prefixes that precede the field sections or content. This reduces the overall
-message size. A message that is truncated at any other point is invalid; see
+prefixes that precede the field sections or content. Truncation can be used to
+reduce the size of messages that have no content for subsequent field sections
```

Comment:
Reusing the word "content" twice to mean different things in one sentence isn't ideal.